dagmc_stats.py

```diff
@@ -270,3 +270,35 @@ def get_area_triangle(my_core, meshset, geom_dim):
         area.append(s)
 
     return area
+
+
+def get_coarseness(my_core,meshset,entity_ranges):
+    """
+    Gets the coarseness of area
+
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    meshset : a meshset containing a certain part of the mesh
+    entity_ranges : the surface entities
+
+    outputs
+    -------
+    coarseness : (list) the coarseness for all surfaces in the meshset
+    """
+    t_p_s = {}
+    coarseness = []
+    
+    for surface in entity_ranges:
+        t_p_s[surface] = my_core.get_entities_by_type(
+                                 surface, types.MBTRI)
+        surf_area = 0
+        for tri in t_p_s[surface]:	
+            side_lengths = get_tri_side_length(my_core, tri)
+            # sqrt(s(s - a)(s - b)(s - c)), where s = (a + b + c)/2
+            s = sum(side_lengths)/2
+            s = np.sqrt(s * np.prod(s - side_lengths))
+            surf_area += s
```

Comment:
Rather than recalculate this all again, isn't this just the triangle area calculation that is done already in a different function? Can you call that function to do this work? You will need to pass in a single surface to the triangle area function and get the list of triangle areas for that surface. Something like this might work better:

```
for surf in geometry:
    triangle_areas = get_triangle_area(surf)  # list would return a list of the triangle areas for the surface of interest
    calculated_coarseness = len(triangle_areas) / sum(triangle_areas) # this is equivalent to number of triangles on the  surface / the total area of the surface.

You do the right coarseness calculation in line 302, but I think you could use some functions you already made in your last PR to calculate the area.